Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Solar",
+    platforms: [
+        .iOS(.v8),
+        .watchOS(.v3),
+        .tvOS(.v9),
+        .macOS(.v10_10),
+    ],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "Solar",
+            targets: ["Solar"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(
+            name: "Solar",
+            dependencies: [],
+            path: "Solar"),
+        .testTarget(
+            name: "Solar iOSTests",
+            dependencies: ["Solar"]),
```

Comment:
This test bundle loads a test fixture at runtime, CorrectResults.json. It needs to be added as a bundle resource in order for the unit tests to pass.

Reply:
@1ec5 ah, yes! Will update.